pkg/reconciler/ingress/ingress.go

```diff
@@ -117,6 +119,10 @@ func makeNewIngress(ingress *v1alpha1.Ingress, ingressClass string) *v1alpha1.In
 			for _, path := range rule.HTTP.Paths {
 				defaultPath := path
 				defaultPath.Splits = splits
+				defaultPath.AppendHeaders = map[string]string{
+					"Async-Original-Host": ingress.Name + "." + ingress.Namespace + ".svc." + network.GetClusterDomainName(),
```

Comment:
use `getClusterLocalDomain` here too?

Reply:
@julz I don't see the helper function -- do you have a link?

Reply:
It’s introduced in this commit on L245 and used 15 lines down from here in the other branch of this if/else :)

Reply:
OH. okay... for some reason thought you were talking about the helper function we were discussing from other PR. /facepalm. Thanks!